File: KeepDog/app/src/main/java/com/app/keepdog/BitPicActivity.java
package com.app.keepdog;

import android.content.Intent;
import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.widget.LinearLayout;

import com.bumptech.glide.Glide;
import com.github.chrisbanes.photoview.PhotoView;

import java.util.ArrayList;
import java.util.List;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.content.ContextCompat;
import androidx.recyclerview.widget.DividerItemDecoration;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

public class BitPicActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_bit_pic);

        Intent intent = getIntent();
        int pic = intent.getIntExtra("pic", 0);
        PhotoView photoView = findViewById(R.id.photoview);
        Glide.with(this).load(pic).into(photoView);
    }
}

File: KeepDog/app/src/main/java/com/app/keepdog/BitPicActivity.kt
package com.app.keepdog

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.bumptech.glide.Glide
import com.github.chrisbanes.photoview.PhotoView

class BitPicActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_bit_pic)
        val intent = intent
        val pic = intent.getIntExtra("pic", 0)
        val photoView = findViewById<PhotoView>(R.id.photoview)
        Glide.with(this).load(pic).into(photoView)
    }
}